build.gradle.kts

```diff
@@ -15,11 +15,12 @@ plugins {
     id("com.jfrog.artifactory")
 }
 
-val branch = prop("branch") ?: "git rev-parse --abbrev-ref HEAD".execute(rootDir.absolutePath).lines().last()
+val branch = prop("branch") ?: System.getenv("TRAVIS_BRANCH")
+             ?: Runtime.getRuntime().exec("git rev-parse --abbrev-ref HEAD").inputStream.reader().readText()
 logger.info("On branch $branch")
 
 group = "be.bluexin"
-version = "$branch-".takeUnless { branch == "master" }.orEmpty() + prop("version_number") + "." + prop("build_number")
+version = branch.takeUnless { branch == "master" }.orEmpty() + prop("version_number") + "." + prop("build_number")
```

Comment:
I think keeping some kind of separator between branch (feature) and actual version number is better

Reply:
Oh, that was accidental 😅